apps/web/src/components/quick-start/in-app-onboarding/TriggerNode.tsx

```diff
@@ -57,7 +57,10 @@ export function TriggerNode({ data }: { data: { label: string; email?: string }
 
 function TriggerButton({ setOpened }: { setOpened: (value: boolean) => void }) {
   const [notificationNumber, setNotificationNumber] = useState(1);
-  const { templates = [], loading: templatesLoading } = useTemplates();
+  const { templates = [], loading: templatesLoading } = useTemplates({
+    pageIndex: 0,
+    pageSize: 100,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Could we extract this to a reusable object (constant) somewhere so that all of these invocations refer to the same configuration?‏